sdk/communication/Azure.Communication.Chat/tests/ChatClients/ChatClientsTest.cs

```diff
@@ -90,70 +91,52 @@ public void ThreadCGUD_MemberAUR_MessageGSU_NotificationT()
             #endregion Snippet:Azure_Communication_Chat_Tests_E2E_GetChatThreadsInfo
             var threadsCount = threads.Count();
 
-            string messageContent = "Let's meet at 11am";
             #region Snippet:Azure_Communication_Chat_Tests_E2E_SendMessage
             SendChatMessageResult sendChatMessageResult = chatThreadClient.SendMessage("Let's meet at 11am");
             #endregion Snippet:Azure_Communication_Chat_Tests_E2E_SendMessage
-            var messageContent2 = "Content for message 2";
-            SendChatMessageResult sendChatMessageResult2 = chatThreadClient2.SendMessage(messageContent2, ChatMessagePriority.High, displayNameMessage);
-            var messageContent3 = "Content for message 3";
-            SendChatMessageResult sendChatMessageResult3 = chatThreadClient3.SendMessage(messageContent3, ChatMessagePriority.High, displayNameMessage);
-            var messageContent4 = "Content for message 4";
-            SendChatMessageResult sendChatMessageResult4 = chatThreadClient3.SendMessage(messageContent4, ChatMessagePriority.High, displayNameMessage);
-            var messageContent5 = "Content for message 5";
-            SendChatMessageResult sendChatMessageResult5 = chatThreadClient3.SendMessage(messageContent5, ChatMessagePriority.High, displayNameMessage);
-            var messageContent6 = "Content for message 6";
-            SendChatMessageResult sendChatMessageResult6 = chatThreadClient3.SendMessage(messageContent6, ChatMessagePriority.High, displayNameMessage);
 
             var messageId = sendChatMessageResult.Id;
-            var messageId2 = sendChatMessageResult2.Id;
-            var messageId3 = sendChatMessageResult3.Id;
-            var messageId4 = sendChatMessageResult4.Id;
-            var messageId5 = sendChatMessageResult5.Id;
-            var messageId6 = sendChatMessageResult6.Id;
-
             #region Snippet:Azure_Communication_Chat_Tests_E2E_GetMessage
             ChatMessage message = chatThreadClient.GetMessage(messageId);
             #endregion Snippet:Azure_Communication_Chat_Tests_E2E_GetMessage
-            ChatMessage message2 = chatThreadClient2.GetMessage(messageId2);
-            ChatMessage message3 = chatThreadClient3.GetMessage(messageId3);
-            ChatMessage message4 = chatThreadClient3.GetMessage(messageId4);
-            ChatMessage message5 = chatThreadClient3.GetMessage(messageId5);
-            ChatMessage message6 = chatThreadClient3.GetMessage(messageId6);
+
+            var messagesMapping = new Dictionary<int, (string, ChatMessage)>
```

Comment:
seems that this dictionary is more like an array, indices are 1 to 6. Also the `string` in `(string, ChatMessage)` is the id of the message which is also in the `ChatMessage` object as well, right? Then why is not it just an array of `ChatMessage`?

Reply:
you are right, now that we are doing together send and get, there is no need to keep message Id as a separate look up. Will use an array and its position to validate.